opencti-platform/opencti-front/src/utils/filters/filtersManageStateUtil.ts

```diff
@@ -0,0 +1,90 @@
+import { Filter, FilterGroup } from './filtersUtils';
```

Comment:
This file could be a good candidate to start making unit testing in front side

Reply:
(This remark is more a global thought than a request to you to make those tests 😅)

Reply:
Every utils files as they are pure function are good candidate IMO :).
We can get check together how we should handle the test in Front.

Reply:
Let's talk about it in the next test session.